packages/core/android/src/main/kotlin/com/datadog/reactnative/DdLogs.kt

```diff
@@ -201,4 +236,8 @@ class DdLogs(reactContext: ReactApplicationContext, logger: Logger? = null) :
         )
         promise.resolve(null)
     }
+
+    companion object {
+        private const val SDK_NOT_INITIALIZED_MESSAGE = "Log sent before SDK init"
```

Comment:
~~Maybe we can add some explanation here? Like "make sure you have called initialize method", etc.~~

Ok, seems this message is not shown to the user, so we can use short one indeed.

Reply:
I can use a message that looks a bit more like an internal message, so it's clear it's never seen by the user 👍